README.md

```diff
@@ -1,2 +1,12 @@
 # Operational-Planning-Tool
-That is a Cloud Infrastructure Project for a Operational Planning of Companies
```

Comment:
This  is a Cloud Infrastructure Project for the operational planning for companies.